docs/vsphere_ci/README.md

```diff
@@ -14,7 +14,7 @@ to be executed from the bastion host or any instance that has access to the vSph
 
 ## Installing Packer
 
-In case of our vSphere environment, we need to install Packer 1.6.6 on the bastion host. The installation steps are:
+In case of our vSphere environment, we need to install Packer 1.6.6+ on the bastion host. The installation steps are:
```

Comment:
I don't think this is needed here. What Packer version did you test? 

Relaxing the Packer version with a lower version constrain may have other implications. For instance, we should point the installation details to a generic page in Packer documentation, instead of [suggesting](https://github.com/openshift/windows-machine-config-operator/blob/912e92a5f09dcd7925488e8958b687dee416454e/docs/vsphere_ci/README.md?plain=1#L20) a fixed version.

Reply:
> instead of suggesting a fixed version

I added the `+` to this line for this reason. We tested with 1.6.6 in the past (and I used it for this golden image too) but there are newer released versions of Packer. I can remove the version suggestion entirely if that'd be better?